Lib/test/datetimetester.py

```diff
@@ -1588,6 +1589,16 @@ def test_strftime(self):
         self.assertEqual(t.strftime(""), "") # SF bug #761337
         self.assertEqual(t.strftime('x'*1000), 'x'*1000) # SF bug #1556784
 
+        # SF bug #137165
```

Comment:
```suggestion
        # See gh-137165
```
I assume you copied this from above, but "SF" means SourceForge, which we no longer use.